vm-virtio/src/net.rs

```diff
@@ -505,17 +498,21 @@ impl Net {
             config_space = Vec::new();
         }
 
-        Ok(Net {
+        let net_backend = VirtioBackend {
             kill_evt: None,
-            pause_evt: None,
-            tap: Some(tap),
             avail_features,
             acked_features: 0u64,
-            config_space,
+            config_space: config_space.clone(),
```

Comment:
Why do you need to clone it?

Reply:
There is compiling error previously if not call clone(), but it does not failed this time after remove clone(), not sure why. Anyway, remove it.